com.unity.formats.alembic/Runtime/Scripts/Importer/Behaviours/AlembicCustomData.cs

```diff
@@ -15,6 +17,37 @@ public class AlembicCustomData : MonoBehaviour
         /// </summary>
         public List<string> FaceSetNames => faceSetNames;
 
+        public struct V2FAttribute
```

Comment:
This should probably be renamed to VertexAttributes and should include a Type field.
The data should be read from c++ as a NativeArray<byte> but presented to the user as a type safe NativeArray<Vector2-3-4> according to the property Type.